index.html

```diff
@@ -377,15 +377,15 @@ <h2 class="">Example of Outline Button</h2>
                     </div>
                     <div class="docs-info">
                         <div class="docs-type">
-                            <h2 class="">Examples of link and icon Button</h2>
+                            <h2>Examples of link and icon Button</h2>
                         </div>
                         <div class="docs-explanation">
                             <p class="text-quaternary">
                                 For icon include <span class="text-highlight">btn-icon icon</span> class.
-                                <br> For link include <span class="text-highlight">btn-primary </span> class or other classes inside <span class="text-highlight"> < a class=""></span> tag.
+                                <br> For link include <span class="text-highlight">btn-primary </span> class or other classes inside <span class="text-highlight"> < a></span> tag.
                             </p>
                         </div>
-                        <div class="docs-example flex-row">
+                        <div class="docs-example flex-row ">
                             <button class="btn-icon icon"><i class="fas fa-shopping-cart"></i> 
```

Comment:
could have given style of button to link instead of having 2 tags seperate for button and link

Reply:
i created btn for both link and button